policy/README.md

```diff
@@ -0,0 +1,353 @@
+# Mobility Data Specification: Policy
+
+This specification contains a collection of RESTful APIs used to specify the digital relationship between *mobility as a service* Providers and the Agencies that regulate them.
+
+* Authors: LADOT
+* Date: 15 May 2019
+* Version: alpha
+
+## Table of Contents
+* [Audience](#audience)
+* [Background](#background)
+* [Distribution](#distribution)
+* [Schema](#schema)
+* [Endpoints](#endpoints)
+* [Authoring](#authoring)
+* [Compliance](#compliance)
+* [Open Topics](#open-topics)
+* [Examples](#examples)
+
+<a name="audience"></a>
+## Audience
+
+There are three intended audiences of this API: 
+
+1. Agencies, as consumers in the form of automated compliance measurement, and as creators of policy
+2. Providers, so that they can measure their compliance status independently
+3. Tooling companies that want to create applications for policy-creation and editing
+
+<a name="background"></a>
+## Background
+
+The goal of this specification is to enable Agencies to create, revise, and publish machine-readable micromobility policy, as set of rules for individual and collective micromobility device behavior exhibited by both providers and riders. Examples of policies include:
+
+* City-wide and localized caps (e.g. “Minimum 500 and maximum 3000 scooters within city boundaries”)
+* Exclusion zones (e.g. “No scooters are permitted in this district on weekends”)
+* Cap allowances (e.g. “Up to 500 additional scooters are permitted near train stations”)
+* Speed-limit restrictions (e.g. “15 mph outside of downtown, 10 mph downtown”)
+* Idle-time and disabled-time limitations (e.g. “5 days idle while rentable, 12 hours idle while unrentable, per device”)
```

Comment:
I think it would be important to add incentive zones to this distribution, as preferred drop offs and preferred pick up zones are crucial

Reply:
Good point.  Working on it.

Reply:
Example included